addons/mail/static/src/model/record_list.js

```diff
@@ -219,7 +219,11 @@ export class RecordListInternal {
         if (inverse && inv) {
             // special command to call addNoinv/deleteNoInv, to prevent infinite loop
             const target = isRecord(val) && val._raw === val ? val._proxy : val;
-            target[inverse] = [[mode === "ADD" ? "ADD.noinv" : "DELETE.noinv", recordList._.owner]];
+            if (target) {
```

Comment:
I'm not sure if it can happen, but it's safer to test the key. 
```suggestion
            if (target && target[inverse]) {
```